en/docs/reference/connectors/kafka-connector/setting-up-kafka.md

```diff
@@ -20,6 +20,9 @@ Copy the following additional client libraries to the `<MI_HOME>/lib` directory
 * [common-utils-5.4.0.jar](https://mvnrepository.com/artifact/io.confluent/common-utils/5.4.0)
 * [kafka-avro-serializer-5.3.0.jar](https://mvnrepository.com/artifact/io.confluent/kafka-avro-serializer/5.3.0)
 * [kafka-schema-registry-client-5.3.0.jar](https://mvnrepository.com/artifact/io.confluent/kafka-schema-registry-client/5.3.0)
+   
+!!! Note "Library versions"
+    Note that the libraries mentioned above are the versions **tested** with the connector.
```

Comment:
after following the steps only, they will see this note. We need to tell the user before hand that following the versions we've tested, giving a hint that they can use whatever they want.

Reply:
Done

Reply:
Shall we further it to say, you can use any compatible version after local testing